sound/soc/sof/topology.c

```diff
@@ -1531,10 +1531,9 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 	/* check token parsing reply */
 	if (ret < 0) {
 		dev_err(scomp->dev,
-			"error: failed to add widget id %d type %d name : %s stream %s\n",
-			tw->shift, swidget->id, tw->name,
-			strnlen(tw->sname, SNDRV_CTL_ELEM_ID_NAME_MAXLEN) > 0
-				? tw->sname : "none");
+			"error: failed to add widget type %d name : %s stream %s\n",
+			swidget->id, tw->name, strnlen(tw->sname, SNDRV_CTL_ELEM_ID_NAME_MAXLEN) > 0
+							? tw->sname : "none");
```

Comment:
The commit message doesn't really describe the changes.

The initial code used tw->shift for the id, but now sdwidget->id is shown as a widget type?

it's likely yet another of those cases where people used 'id' for different things...

Reply:
Ack, to use tw->shift for widget id is a mistake in the current code, so this patch removes it.
swidget->id is defined as DAPM widget type, should I change the line to "failed to add widget of DAPM type %d .."?